packages/node/src/integrations/utils/http.ts

```diff
@@ -48,7 +48,8 @@ export function extractUrl(requestOptions: RequestOptions): string {
     !requestOptions.port || requestOptions.port === 80 || requestOptions.port === 443 ? '' : `:${requestOptions.port}`;
   // do not include search or hash in span descriptions, per https://develop.sentry.dev/sdk/data-handling/#structuring-data
   const path = requestOptions.pathname || '/';
-  const authority = requestOptions.auth ? `${requestOptions.auth}@` : '';
+  // // always filter authority, see https://develop.sentry.dev/sdk/data-handling/#structuring-data
+  const authority = requestOptions.auth ? '[Filtered]:[Filtered]@' : '';
```

Comment:
```suggestion
  const authority = requestOptions.auth ? '[Filtered]:[Filtered]@' : '';
```

This always adds a `[Filtered]` for both, username and password, even if only one is present. We can refactor this to only set the respective one but I'd argue it's safer from a PII/user identification view to just always show both.